virtual_rainforest/models/animals/carcasses.py

```diff
@@ -0,0 +1,17 @@
+"""The 'carcass_module.
+
+This file provides toy carcass module functionality for developing the animal and soil
+modules.
+"""
```

Comment:
This should also follow the 'no header' style we are using across the other files - the reasons for this are basically just to avoid having repeated headers / floating single line paragraphs in the documentation.

```suggestion
"""The 'carcass` module provides toy carcass module functionality for developing the animal and soil
modules.
"""  # noqa: #D205, D415
```